### PR TITLE
Larch flush

### PR DESCRIPTION
**Pull request by jcorbin**
r @Raynos, @rf

### CODE REVIEW

**Comment by jcorbin** [comment]
@Raynos where should we add call site to the new logtron logger.close?

**Comment by rf** [comment]
:+1:

The logtron backend should have a destroy method, that can close logtron. So `clients.js` will call `Larch#destroy` -> `ReservoirBackend#destroy` -> `LogtronBackend#destroy` -> `Logtron#close`

File: hyperbahn/package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "is-empty-object": "1.0.0",
     "json-stringify-safe": "5.0.0",
     "leaked-handles": "5.1.0",
-    "logtron": "8.3.4",
+    "logtron": "8.10.2",
```
Comment (Raynos) [inline hyperbahn/package.json]: I hope we dont break kafka / search / etc in these 7 minor versions ...

**Comment by Raynos** [comment]
lgtm.

**Pull request merged**